### DISCUSSION TITLE
CSV FORMAT

### DISCUSSION THREAD

**Discussion by deiivid-88** [hneemann/Digital#1343] [Q&A]
Hi,
I am looking at importing a csv file, what is the representation of don't care term in the csv file?. I've tried with 'x' , 'X', '2', and none of them work. 
Thanks,

**Comment by hneemann** [comment]
see #1344

**Comment by hneemann** [comment]
I must have overlooked the function and nobody has noticed it yet.

**Comment by hneemann** [comment]
You can try v0.31-4-gec82bfc found [here](https://infdigital.dhbw-mosbach.de/).
